tests/fixtures/with-lock-file/composer.lock

```diff
@@ -8,20 +8,24 @@
     "packages": [
         {
             "name": "ehime/hello-world",
```

Comment:
The changes to this lock file need to be undone as this is explicitly testing that a non-latest locked version of the `hello-world` package will be downloaded, so updating this lock file means that's no longer being tested.